cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaExporter.java

```diff
@@ -0,0 +1,296 @@
+        return portList;
+    }
+
+    public Deployment generateDeployment(boolean isOpenShift, ImagePullPolicy imagePullPolicy, List<LocalObjectReference> imagePullSecrets) {
+        DeploymentStrategy updateStrategy = new DeploymentStrategyBuilder()
+                .withType("Recreate")
+                .build();
+
+        return createDeployment(
+                updateStrategy,
+                Collections.emptyMap(),
+                Collections.emptyMap(),
+                getMergedAffinity(),
+                getInitContainers(imagePullPolicy),
+                getContainers(imagePullPolicy),
+                Collections.emptyList(),
+                imagePullSecrets
+        );
+    }
+
+    @Override
+    protected List<Container> getContainers(ImagePullPolicy imagePullPolicy) {
+
+        List<Container> containers = new ArrayList<>();
+
+        Container container = new ContainerBuilder()
+                .withName(name)
+                .withImage(getImage())
+                .withCommand("/opt/kafka-exporter/kafka_exporter_run.sh")
+                .withEnv(getEnvVars())
+                .withPorts(getContainerPortList())
+                .withLivenessProbe(ModelUtils.createHttpProbe(livenessPath, METRICS_PORT_NAME, livenessProbeOptions))
+                .withReadinessProbe(ModelUtils.createHttpProbe(readinessPath, METRICS_PORT_NAME, readinessProbeOptions))
+                .withResources(getResources())
+                .withImagePullPolicy(determineImagePullPolicy(imagePullPolicy, getImage()))
+                .build();
+
+        containers.add(container);
+
+        return containers;
+    }
+
+    @Override
+    protected List<EnvVar> getEnvVars() {
+        List<EnvVar> varList = new ArrayList<>();
+
+        varList.add(buildEnvVar(ENV_VAR_KAFKA_EXPORTER_LOGGING, logging));
+        varList.add(buildEnvVar(ENV_VAR_KAFKA_EXPORTER_KAFKA_VERSION, version));
+        varList.add(buildEnvVar(ENV_VAR_KAFKA_EXPORTER_GROUP_REGEX, groupRegex));
+        varList.add(buildEnvVar(ENV_VAR_KAFKA_EXPORTER_TOPIC_REGEX, topicRegex));
+        varList.add(buildEnvVar(ENV_VAR_KAFKA_EXPORTER_KAFKA_SERVER, KafkaCluster.serviceName(cluster) + ":9092"));
+        varList.add(buildEnvVar(ENV_VAR_KAFKA_EXPORTER_ENABLE_SARAMA, String.valueOf(saramaLoggingEnabled)));
+
+        varList.add(buildEnvVar(ENV_VAR_STRIMZI_LIVENESS_PERIOD,
+                String.valueOf(livenessProbeOptions.getPeriodSeconds() != null ? livenessProbeOptions.getPeriodSeconds() : DEFAULT_HEALTHCHECK_PERIOD)));
+        varList.add(buildEnvVar(ENV_VAR_STRIMZI_READINESS_PERIOD,
+                String.valueOf(readinessProbeOptions.getPeriodSeconds() != null ? readinessProbeOptions.getPeriodSeconds() : DEFAULT_HEALTHCHECK_PERIOD)));
+
+        return varList;
+    }
+
+    /**
+     * Generates the PodDisruptionBudget.
+     *
+     * @return The PodDisruptionBudget.
+     */
+    public PodDisruptionBudget generatePodDisruptionBudget() {
```

Comment:
Do we actually use / want to use a PodDisruptionBudget here? If yes, I didn't saw the template. But I think we do not need it here.